packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -171,6 +171,22 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
     IERC20(registry.getAddressForOrDie(STABLE_TOKEN_REGISTRY_ID)).transfer(to, value);
   }
 
+  /**
+   * @notice Wrapper function for any ERC-20 transfer function.
+   * @dev Protects against celo/locked celo balance changes.
+   */
+  function genericTransfer(address erc20, address to, uint256 value)
+    external
+    onlyWhenInProperState
+  {
+    uint256 balanceBefore = address(this).balance +
+      getLockedGold().getTotalLockedGold(address(this));
+    IERC20(erc20).transfer(to, value);
```

Comment:
Would technically a "flash self-loan" of non released CELO be still possible here? I.e. `to` is a contract that receives CELO, uses it to make a profit, then returns it back here. Or is the fact that `transfer` limits gas enough of a defense against this?

Either way, would just checking that `erc20` is not the registered GoldToken address be more explicit?